### PR TITLE
Stop calling inductive indices "annotations" in the doc

### PR DESCRIPTION
**Pull request by SkySkimmer**
cf also #16762

### CODE REVIEW

File: doc/sphinx/language/core/inductive.rst [hunk 1]
```diff
@@ -1084,7 +1084,7 @@ Conversion is preserved as any (partial) instance :math:`I_j~q_1 … q_r` or
 .. flag:: Auto Template Polymorphism
 
    This :term:`flag`, enabled by default, makes every inductive type declared
-   at level :math:`\Type` (without annotations or hiding it behind a
+   at level :math:`\Type` (without explicit universe instance or hiding it behind a
```
Comment (Alizter) [inline doc/sphinx/language/core/inductive.rst:1087]:
```suggestion
   at level :math:`\Type` (without an explicit universe instance or hiding it behind a
```

**Review by Alizter — APPROVED** [review]

File: doc/sphinx/language/core/inductive.rst [hunk 2]
```diff
@@ -247,7 +247,7 @@ the same parameter values of its specification.
      because the conclusion of the type of constructors should be :g:`listw A`
      in both cases.
 
-   + A parameterized inductive definition can be defined using annotations
+   + A parameterized inductive definition can be defined using indices
```
Comment (jfehrle) [inline doc/sphinx/language/core/inductive.rst:250]: What about the uses of "annotation" on lines 420, 423, 445 and 446?

**Review by jfehrle — COMMENTED** [review]
LGTM.  One question below.  Also, universe annotations should still be "annotations"?

File: doc/sphinx/language/core/inductive.rst (hunk 2 above)
Comment (SkySkimmer) [reply doc/sphinx/language/core/inductive.rst:250]: Those are about the `{struct _}` on fixpoints, I don't think they have a name other than annotation.

"Universe annotation" could be "universe instance" instead or maybe would need to be "explicit universe instance".

**Comment by SkySkimmer** [comment]
@coqbot run full ci

**Comment by Alizter** [comment]
@coqbot merge now

failures look unrelated

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**